history/members.json

```
@@ -176,7 +176,7 @@
 			{
 				"start": "2004-02-01",
 				"original_end": "2006-01-31",
-				"end": "2004-07-11",
+				"end": "2004-06-10",
```

Looks like this should be 
```suggestion
				"end": "2004-06-11",
```
?

---

Yes, indeed. Fixed.